admin-dev/themes/new-theme/js/components/entity-search-input.ts

```diff
@@ -309,6 +308,20 @@ export default class EntitySearchInput {
     }
   }
 
+  private showSuggestion(entity: any): string {
+    if (!isUndefined(this.options.suggestionTemplate)) {
+      return this.options.suggestionTemplate(entity);
+    }
+
+    let entityImage = '';
+
+    if (Object.prototype.hasOwnProperty.call(entity, 'image')) {
```

Comment:
nitpick but 
```suggestion
    if (entity.image) {
```

Should do the job, I think I've never used hasOwnProperty in my life

Reply:
It's ESlint which forces to do this 🤷‍♂️